packages/ckeditor5-engine/docs/_snippets/framework/extending-content-custom-element-converter.js

```diff
@@ -123,19 +128,51 @@ function insertViewElements( data, conversionApi, infoBox, infoBoxTitle, infoBox
 		infoBoxContent
 	);
 
-	// The default mapping between the model <infoBox> and its view representation.
+	// The mapping between the model <infoBox> and its view representation.
 	conversionApi.mapper.bindElements( data.item, infoBox );
-	// However, since the model <infoBox> content needs to end up in the inner
-	// <div class="info-box-content">, you need to bind one with another overriding
-	// a part of the default binding.
-	conversionApi.mapper.bindElements( data.item, infoBoxContent );
 
 	conversionApi.writer.insert(
 		conversionApi.mapper.toViewPosition( data.range.start ),
 		infoBox
 	);
 }
 
+function createModelToViewPositionMapper( view ) {
+	return ( evt, data ) => {
+		const modelPosition = data.modelPosition;
+		const parent = modelPosition.parent;
+
+		if ( !parent.is( 'element', 'infoBox' ) ) {
+			return;
+		}
+
+		const viewElement = data.mapper.toViewElement( parent );
+		const viewContentElement = findContentViewElement( view, viewElement );
+
+		data.viewPosition = data.mapper.findPositionIn( viewContentElement, modelPosition.offset );
+	};
+}
+
+function findContentViewElement( editingView, viewElement ) {
+	for ( const value of editingView.createRangeIn( viewElement ) ) {
```

Comment:
It's sad that we don't have an easier API to find a node inside another node :( Also worth reporting in the domain:dx, squad:dx (although, we may have it reported already).

Reply:
#9465